net/HttpRequest.hpp

```diff
@@ -1397,7 +1397,7 @@ class Session final : public ProtocolHandlerInterface
 
         std::shared_ptr<StreamSocket> socket = _socket.lock();
         if (socket)
-            socket->dumpState(os);
+            os << indent << "socket: #" << socket->getFD();
```

Comment:
Should we ditch the workaround of https://github.com/CollaboraOnline/online/commit/7769161afb19f18c484f90931bce0fc00c041dcc as a follow up?

Reply:
Yes. I didn't remove it just yet until we all agree it's safe.